conf/scalardb-graphql-custom-values.yaml

```diff
@@ -41,31 +41,35 @@ grafanaDashboard:
 
 tolerations:
   - effect: NoSchedule
-    key: kubernetes.io/app
+    key: scalar-labs.com/dedicated-node
     operator: Equal
-    value: scalardbgraphqlpool
+    value: scalardb
 
-resources: {}
+resources:
+  requests:
+    cpu: 2000m
+    memory: 4Gi
+  limits:
+    cpu: 2000m
+    memory: 4Gi
 
 affinity:
   nodeAffinity:
     requiredDuringSchedulingIgnoredDuringExecution:
       nodeSelectorTerms:
         - matchExpressions:
-            - key: agentpool
+            - key: scalar-labs.com/dedicated-node
               operator: In
               values:
-                - scalardbgraphqlpool
+                - scalardb
```

Comment:
I think that we expect the `Scalar DB Server` and `Scalar DB GraphQL Server` to be deployed on the same worker node.
So, I set the same label as Scalar DB Server in the custom values file of Scalar DB GraphQL Server.